test/async-hooks/test-timers.setInterval.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+const initHooks = require('./init-hooks');
+const { checkInvocations } = require('./hook-checks');
+const TIMEOUT = common.platformTimeout(100);
+
+const hooks = initHooks();
+hooks.enable();
+
+// install first timeout
```

Comment:
Nit: please always use capital letters for the first character of a comment and use punctuation.

Reply:
Was left over from another test for async hooks. It's not a very helpful comment to be honest. Removing.